editor/script_editor_debugger.cpp

```diff
@@ -1646,7 +1646,7 @@ void ScriptEditorDebugger::_error_selected(int p_idx) {
 		md.push_back(st[i + 1]);
 		md.push_back(st[i + 2]);
 
-		String str = func + " in " + script.get_file() + ":line " + itos(line);
+		String str = func + (line != NULL ? (" in " + script.get_file() + ":line " + itos(line)) : "");
```

Comment:
This is comparing line to 0. The script debugger handles lines as beginning with 1, so using 0 is fine here. I would use the script path for this though. Something like this:

```
String str = func;
if (script.length()) {
    str += " in " + script.get_file();
    if (line != 0) // do not show _":line 0"_ if the line if it's unknown
        str += ":line " + itos(line);
}
```

Reply:
I changed this, and took the opportunity to do the same thing with the tooltip.